week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -126,6 +126,9 @@ function getLocationName(a) {
   Advanced challange: try to use arrow function when invoking an array method.
 */
 function journeyPlanner(locations, transportMode) {
+  let checkTransport = locations.filter((i) => i.includes(transportMode));
+  let result = checkTransport.map((i) => i[0]);
+  return result;
```

Comment:
This is correct but the letters i and j are normally for indexes, for example in for loops where you would have, for example 'i = 0; i < 5; i++'. It is usually better to use a more meaningful word.
Look at the solution here https://github.com/anthonytranDev/week-3-solutions/blob/master/mandatory/5-journey-planner.js lines 146-155.